package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A monorepo for Safe modules",
   "main": "dist/index.js",
   "workspaces": [
-    "modules/*",
     "packages/*",
+    "modules/*",
     "examples/*"
```

Comment:
Its LOL, but package ordering matters for the order that scripts are executed :facepalm: